### PR TITLE
SEO-1908-MKW-WF-Scheduler-masterdocs-Onpage-interlink

### PR DESCRIPTION
**Pull request by Gowri-V-L**
Hi All,

Content task :https://syncfusion.atlassian.net/browse/CONTENT-5748
Could please review and merge the below PR's

### CODE REVIEW

File: WindowsForms/Scheduler/Getting-Started.md [hunk 1]
```diff
@@ -94,15 +94,15 @@ Follow the steps to create an appointment:
...
 
-This file defines several classes that implements  the ScheduleControl interfaces that manages the data associated with appointments appeared in the calendar. These interfaces are discussed in detail later in this UserGuide. 
+This file defines several classes that implements the ScheduleControl interfaces that manages the data associated with appointments appeared in the calendar. These interfaces are discussed in detail later in this UserGuide. 
 
-Use the implementation provided in the `SimpleScheduleDataProvider.cs` file. This file ships as part of the [Schedule sample](https://github.com/syncfusion/winforms-demos/tree/master/schedulecontrol/Scheduler%20Demo/CS). Drill down to this folder and add this file to your project by using the Solution Explorer window.
+Use the implementation provided in the `SimpleScheduleDataProvider.cs` file. This file ships as part of the [winforms scheduler sample](https://github.com/syncfusion/winforms-demos/tree/master/schedulecontrol/Scheduler%20Demo/CS). Drill down to this folder and add this file to your project by using the Solution Explorer window.
```
Comment (Girija-Rajendran) [inline WindowsForms/Scheduler/Getting-Started.md:99]: Platform and control names should be in capital letters.
[WinForms Scheduler sample].......

File: WindowsForms/Scheduler/Getting-Started.md [hunk 2]
```diff
@@ -179,11 +179,11 @@ End Namespace
 
 4. Press `F5` key to compile and run your application. 
 
-![Adding appointment in schedule control](Getting-Started_images/Getting-Started_img14.jpeg)
+![Adding appointment in WinForms schedule control](Getting-Started_images/Getting-Started_img14.jpeg)
```
Comment (Girija-Rajendran) [inline WindowsForms/Scheduler/Getting-Started.md:183]: ....WinForms Scheduler...

File: WindowsForms/Scheduler/Getting-Started.md [hunk 3]
```diff
@@ -748,6 +748,8 @@ End Property
 {% endtabs %}
 ## Recurrence appointment
 
-The Schedule control supports creating the recurring appointment. By this recurrence appointment, you can process on the daily, weekly, monthly, or yearly view to create the recurrence rule with or without end date. Details are explained in the following link:
+The C# WinForms Calendar Scheduler control supports creating the recurring appointment. By this recurrence appointment, you can process on the daily, weekly, monthly, or yearly view to create the recurrence rule with or without end date. Details are explained in the following link:
 
-[Create the recurrence appointment](time-interval)
+[Create the recurrence appointment in WinForms Calendar Scheduler control ](time-interval)
+
+N> You can also explore our [WinForms Scheduler example](https://github.com/syncfusion/winforms-demos/tree/master/schedulecontrol) to know how to schedule and manage appointments through an intuitive user interface, similar to the Outlook calendar.
```
Comment (Girija-Rajendran) [inline WindowsForms/Scheduler/Getting-Started.md:755]: .. that shows how to schedule and......

File: WindowsForms/Scheduler/Overview.md [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
+The Syncfusion Scheduler is a Windows Forms class library built around the functionalities found in the Windows Forms Grid control. The control allows you to add scheduling support to your applications. 
 
-The scheduling support includes creating new appointments, displaying those appointments in a variety of views, including Monthly, Daily, Weekly, Work Week, and multiple days. In the daily formats, you can use the UI to drag appointments to another time slot and to extend appointments. A flexible navigation calendar lets you easily home on the dates you would like to see in the Schedule control. 
+The most popular WinForms Scheduler support includes creating new appointments, displaying those appointments in a variety of views, including Monthly, Daily, Weekly, Work Week, and multiple days. In the daily formats, you can use the UI to drag appointments to another time slot and to extend appointments. A flexible navigation calendar lets you easily home on the dates you would like to see in the Schedule control.
```
Comment (Girija-Rajendran) [inline WindowsForms/Scheduler/Overview.md:14]: get content review for this

File: WindowsForms/Scheduler/Getting-Started.md (hunk 1 above)
Comment (Gowri-V-L) [reply WindowsForms/Scheduler/Getting-Started.md:99]: changed

File: WindowsForms/Scheduler/Overview.md (hunk 4 above)
Comment (Gowri-V-L) [reply WindowsForms/Scheduler/Overview.md:14]: modified

File: WindowsForms/Scheduler/Getting-Started.md (hunk 2 above)
Comment (Gowri-V-L) [reply WindowsForms/Scheduler/Getting-Started.md:183]: modified

File: WindowsForms/Scheduler/Overview.md [hunk 5]
```diff
@@ -30,3 +30,5 @@ The Scheduler control finds a wide variety of applications such as Time Tables,
 * Globalization and localization: Supports localized static text, day, and month names based on the culture.
 * Recurrence appointment: Supports schedule recurring appointments to repeat daily, weekly, monthly, or yearly.
 * Touch support: Completely supports swiping, panning, zooming, and more.
+
+N> You can also explore our [WinForms Scheduler example](https://github.com/syncfusion/winforms-demos/tree/master/schedulecontrol) to know how to schedule and manage appointments through an intuitive user interface, similar to the Outlook calendar.
```
Comment (Girija-Rajendran) [inline WindowsForms/Scheduler/Overview.md:34]: You can also explore our [WinForms Scheduler example](https://github.com/syncfusion/winforms-demos/tree/master/schedulecontrol) that shows......

Comment (Gowri-V-L) [reply WindowsForms/Scheduler/Overview.md]: changed

**Review by Girija-Rajendran — APPROVED** [review]

**Review by ChristoIssac — APPROVED** [review]

**Pull request merged**